frontend/src/app/AppRoutes.tsx

```diff
@@ -31,21 +32,25 @@ const AppRoutes: React.FC = () => {
       fallback={<ApplicationsPage title="" description="" loaded={false} empty={true} />}
     >
       <Routes>
-        <Route path="/" element={<InstalledApplications />} />
-        <Route path="/explore" element={<ExploreApplications />} />
-        <Route path="/resources" element={<LearningCenterPage />} />
-        {isAllowed && <Route path="/projects/*" element={<ProjectViewRoutes />} />}
-        {isAllowed && <Route path="/notebookController/*" element={<NotebookController />} />}
-        {isAllowed && (
-          <Route
-            path="/notebook/:namespace/:notebookName/logout"
-            element={<NotebookLogoutRedirectPage />}
-          />
+        {isAllowed ? (
```

Comment:
Can you avoid doing large conditionals for a single point -- Wouldn't it work if we just use the exit early pattern? I don't think we _need_ the ApplicationPage fallback either or even the suspense as there is no logic here.